puppet/modules/web/files/deploy-yumrepo.sh

```diff
@@ -93,15 +96,21 @@ REPO_SOURCE=$1
 REPO_DEST=$2
 OVERWRITE=${3:-false}
 MERGE=${4:-false}
+STAGING=${5:-false}
 
 if [[ -z $REPO_SOURCE ]] || [[ -z $REPO_DEST ]] ; then
 	echo "Usage: $0 REPO_SOURCE REPO_DEST OVERWRITE MERGE"
 	exit 1
 fi
 
-REPO_SOURCE_BASE="rsync://koji.katello.org/releases"
-REPO_SOURCE_RPM="${REPO_SOURCE_BASE}/yum/${REPO_SOURCE}"
-REPO_SOURCE_SRPM="${REPO_SOURCE_BASE}/source/${REPO_SOURCE}"
+if [[ $STAGING == true ]] ; then
+	REPO_SOURCE_BASE="rsync://rsync.theforeman.org/stagingyum/"
```

Comment:
https://github.com/theforeman/foreman-infra/pull/1878 to actually deploy that part ;-)

Reply:
This appears to have been merged and deployed. Thanks!